openedx/core/djangoapps/user_api/tests/test_views.py

```diff
@@ -1070,14 +1070,19 @@ def test_extension_form_fields(self):
         )
 
     @ddt.data(
-        ('pk', 'PK'),
-        ('Pk', 'PK'),
-        ('pK', 'PK'),
-        ('PK', 'PK'),
-        ('us', 'US'),
+        ('pk', 'PK', 'Bob123', 'Bob123'),
```

Comment:
Do you need to add an entry to cover the null username scenario from [L268](https://github.com/edx/edx-platform/pull/15929/files#diff-fe2e2bebef5c5b8dc202c1768d240c6eR268)?